compile.py

```diff
@@ -0,0 +1,379 @@
+#!/bin/python3
+#
+# Copyright (C) 2023 Savoir-faire Linux, Inc.
```

Comment:
Who has chosen the license?
In the other project, we have dual GPLv3 and Apache license.